apps/ewallet_db/lib/ewallet_db/helpers/crypto.ex

```diff
@@ -4,7 +4,7 @@ defmodule EWalletDB.Helpers.Crypto do
   """
   alias Plug.Crypto
 
-  @spec generate_key(integer) :: binary
+  @spec generate_key(Integer.t()) :: binary()
```

Comment:
Would `integer()` be more accurate than `Integer.t()`?

I also couldn't find a type defined on [`Integer`](https://github.com/elixir-lang/elixir/blob/v1.6.5/lib/elixir/lib/integer.ex).